src/main/java/com/vinylteam/vinyl/util/impl/VinylUaParser.java

```diff
@@ -15,52 +17,77 @@
 import java.util.List;
 
 public class VinylUaParser implements VinylParser {
+
+    private final Logger logger = LoggerFactory.getLogger(getClass());
     private final String startLink = "http://vinyl.ua";
+    private final String classContainingGenresLinks = "dropdown-menu dropdown-menu-left";
+    private final String classContainingPagesLinks = "pagination-wrapper margin-top-20";
+    private final String classContainingVinyl = "vinyl-release showcase";
```

Comment:
I think we don`t need fields for these strings. And we shouldn`t log info about them